elfeed-score.el

```diff
@@ -4,6 +4,8 @@
 
 ;; Author: Michael Herstine <sp1ff@pobox.com>
 ;; Version: 0.4.4
+;; Package-Version: 20200328.1855
+;; Package-Commit: 916c47b3590b2ff3c5075dcc1def4b36a4b14947
```

Comment:
Not sure what these are?

Reply:
I think this comes from getting the .el file from MELPA